common/option-model.cpp

```diff
@@ -391,8 +393,9 @@ bool option_model::draw_slider( notifications_model & model,
         }
     }
 
-    ImGui::PushItemWidth( -1 );
-
+    ImGui::PushItemWidth(245);
```

Comment:
Why 245?

Reply:
fixed (returned to default window item width then using ImGui::PushItemWidth( -1 ) )